README.md

```diff
@@ -165,6 +166,12 @@ Conda environments
 [Conda environments](https://www.nextflow.io/docs/latest/conda.html) provide another option for managing software packages in your workflow. 
 
 
+Spack environments
+------------------
+
+[Spack environments](https://www.nextflow.io/docs/latest/spack.html) provide an option to build software packages from source using Spack, a popular package manager within the HPC community.
```

Comment:
```suggestion
Spack environments provide an option to build software packages from source using Spack, a popular package manager within the HPC community.
```

Eventually there will be the link `www.nextflow.io/docs/latest/spack.html`, but it would be better to add this once the final docs is published